save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreateOrganizationView.kt

```diff
@@ -93,9 +87,8 @@ class CreateOrganizationView : AbstractView<Props, OrganizationSaveViewState>(tr
                     post("$apiUrl/organization/save", headers, Json.encodeToString(newOrganizationRequest))
 
             if (responseFromCreationOrganization.ok) {
-                setState {
-                    isToRedirect = true
-                }
+                window.location.href = "${window.location.origin}#/${organizationName.replace(" ", "%20")}/"
+                window.location.reload()
```

Comment:
Why do we need reloading here?

Reply:
It seems that we need to rerender the page so either we need to reload it, or change the state. @akuleshov7 finds it better to reload it.

Reply:
There is also a way to redirect [using react-router api](https://reactrouter.com/docs/en/v6/api#usenavigate) and we should start using it eventually, maybe not in this PR

Reply:
Ok, will take into consideration.